src/main/java/uk/gov/hmcts/darts/audio/service/MediaRequestService.java

```diff
@@ -64,5 +65,6 @@ public interface MediaRequestService {
 
     MediaHideResponse adminHideOrShowMediaById(Integer mediaId, MediaHideRequest mediaHideRequest);
 
-    List<AdminMediaSearchResponseItem> adminMediaSearch(Integer transformedMediaId);
+    List<AdminMediaSearchResponseItem> adminMediaSearch(Integer transformedMediaId, List<Integer> hearingIds, OffsetDateTime startAt,
```

Comment:
Can we take the opportunity to move this (and impl) to `AdminMediaService`/`AdminMediaServiceImpl`, as fundamentally we're looking for `media` objects, rather than `media_request` objects. Maybe worth renaming to remove mention of `search`, as we'll soon have a separate endpoint responsible for advanced searching of media. Suggest `getMedias`/`filterMedias` or something similar.